bin/plotting/pycbc_plot_waveform

```diff
@@ -0,0 +1,159 @@
+from mpl_toolkits.axes_grid1.inset_locator import zoomed_inset_axes
+from mpl_toolkits.axes_grid1.inset_locator import mark_inset
+from pycbc import waveform
+from pycbc import version
+from pycbc import results
+from pycbc.fft import ifft
+from pycbc.types import complex_same_precision_as
+from pycbc.types import TimeSeries, zeros, complex64
+
+parser = argparse.ArgumentParser(usage='',
+    description="Plot a waveform in both time and frequency domain.")
+parser.add_argument('--version', action='version', 
+                    version=version.git_verbose_msg)
+parser.add_argument('--output-file', required=True)
+parser.add_argument("--low-frequency-cutoff", type=float,
+                  help="The low frequency cutoff to use for generation.")
+parser.add_argument("--approximant", type=str,
+                  help="The name of the approximant to use for filtering. "
+                      "Do not use if using --use-params-of-closest-injection.")
+parser.add_argument("--mass1", type=float, required=True,
+                  help="The mass of the first component object. "
+                      "Do not use if using --use-params-of-closest-injection.")
+parser.add_argument("--mass2", type=float, required=True,
+                  help="The mass of the second component object. "
+                      "Do not use if using --use-params-of-closest-injection.")
+parser.add_argument("--spin1z", type=float, required=True,
+                  help="The aligned spin of the first component object. "
+                      "Do not use if using --use-params-of-closest-injection.")
+parser.add_argument("--spin2z", type=float, required=True,
+                  help="The aligned pin of the second component object. "
+                      "Do not use if using --use-params-of-closest-injection.")
+parser.add_argument("--sample-rate", type=float, required=True,
+                  help="Sample rate to use when generating waveform.")
+parser.add_argument("--waveform-length", type=float, required=True,
+                  help="Used to set the value of delta-f when either generating"
+                       "in frequency domain, or when FFTing.")
+# Optional arguments for precession
+parser.add_argument("--spin1x", type=float, default=0,
+                  help="Non-aligned spin of the first component object. ")
+parser.add_argument("--spin1y", type=float, default=0,
+                  help="Non-aligned spin of the first component object. ")
+parser.add_argument("--spin2x", type=float, default=0,
+                  help="Non-aligned spin of the second component object. ")
+parser.add_argument("--spin2y", type=float, default=0,
+                  help="Non-aligned spin of the second component object. ")
+parser.add_argument("--inclination", type=float, default=0,
+                  help="Non-aligned spin of the second component object. ")
+parser.add_argument("--u-val", type=float, default=0,
+                  help="Ratio between h_+ and h_x according to .... . ")
```

Comment:
What's u-val? Can you give a better help message? At least say when you can ignore this.